lib/dockerize_rails/constants.rb

```diff
@@ -17,40 +17,48 @@ module Constants
     DATABASE_HOST_REMOTE = 'remote'.freeze
 
     COMMANDS = {
-      configure: {
+        configure: {
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

<!-- Comment by [SideCI](https://sideci.com) -->